scripts/apidev.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os
+import shutil
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+
+from watchdog.events import FileSystemEventHandler
+from watchdog.observers import Observer
+
+from azul.deployment import aws
+from azul.modules import load_app_module
+from azul.openapi import annotated_specs
+
+SPEC_FILE = 'openapi.json'
```

Comment:
We don't do upper case constants.

Reply:
```diff
Index: scripts/apidev.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- scripts/apidev.py	(revision 7068a20c2848c16d2304c25ffcbd187d239aa565)
+++ scripts/apidev.py	(date 1566351360490)
@@ -1,6 +1,5 @@
 import json
 import os
-import shutil
 from http.server import HTTPServer, SimpleHTTPRequestHandler
 
 from watchdog.events import FileSystemEventHandler
@@ -12,8 +11,6 @@
 
 SPEC_FILE = 'openapi.json'
 parent_dir = os.path.realpath(os.path.dirname(__file__))
-static_dir = os.path.join(parent_dir, 'apidev_static')
-web_page = os.path.join(static_dir, 'swagger-editor.html')
 
 
 def write_specs(gateway_id, app, api_info):
@@ -23,14 +20,17 @@
 
 
 def main():
+    static_dir = os.path.join(parent_dir, 'apidev_static')
+    web_page = os.path.join(static_dir, 'swagger-editor.html')
     service = load_app_module('service')
-
-    server_address = ('localhost', 8787)
-    server_url = f"http://{':'.join(map(str, server_address))}"
-    httpd = HTTPServer(server_address, SimpleHTTPRequestHandler)
+    # That's what made me chuckle earlier today
+    host, port = 'localhost', 8787
+    server_url = f"http://{host}:{port}"
+    httpd = HTTPServer((host, port), SimpleHTTPRequestHandler)
     address = f"{server_url}/{os.path.relpath(web_page)}?url={server_url}/{os.path.relpath(parent_dir)}/{SPEC_FILE}"
     print(f'Open {address} in browser to validate changes.')
-    print('If you changed/added any routes, make sure to deploy!')
+    print('If you changed/added any routes, make sure to `make deploy` afterwards!')
+    # TODO: explain why
 
     gateway_id = aws.api_gateway_id('service', validate=True)
     event_handler = UpdateHandler(service, gateway_id)
```

Reply:
Ahh... Yeah I had just copied that straight out of the HTTPServer docs. Didn't even think to simplify.